app/controllers/api/v1/lightweight_activities_controller.rb

```diff
@@ -5,4 +5,14 @@ def destroy
     activity.destroy
     render :json => { success: true }
   end
+
+  # GET /api/v1/activities/1.json
+  def show
+    activity = LightweightActivity.find(params[:id])
+    lightweight_activity_json = activity.export.to_json
+    respond_to do |format|
+      format.html # show.html.html.haml
+      format.json { render json: lightweight_activity_json }
+    end
```

Comment:
```suggestion
    render json: lightweight_activity_json
```
Since this is an api I don't think it should support the html format.

Reply:
:+1: